pkgs/applications/editors/neovim/default.nix

```diff
@@ -6,6 +6,11 @@
 
 # now defaults to false because some tests can be flaky (clipboard etc)
 , doCheck ? false
+, nodejs ? null, fish ? null
+
+# set devMode to true to pull the software used to develop on neovim
+, devMode ? false
```

Comment:
I think most of the functionality this flag suggests can be coupled with that of `doCheck`, so:
```suggestion
```

Reply:
devMode is really for developers to use. I intend to enable doCheck at some point in the future hence the decorrelation.